app/views/ladder/LadderView.coffee

```diff
@@ -184,7 +184,10 @@ module.exports = class LadderView extends RootView
     return unless @supermodel.finished()
     @$el.toggleClass 'single-ladder', @level.isType 'ladder'
     unless @tournamentState is 'ended'
-      @insertSubView(@ladderTab = new LadderTabView({league: @league, tournament: @tournamentId}, @level, @sessions))
+      if @level.get('slug') in ['sorcerers', 'blazing-battle']
```

Comment:
What is this `slug` check in here for? In the US, we have `'ended'` tournaments for `mages-might` and `infinite-inferno` as well.

Reply:
@nwinter i want to make this leaderboard for codequest only first to check if this leaderboard works fine to customers. Then we can remove this condition once we decide to migrate all leaderboard to new one (and maybe remove old files)

Reply:
and 'blazing-battle' is for testing only since it contains lots of redis caches 😂

Reply:
But aren't we already using the leaderboard in production ([example](https://codecombat.com/play/ladder/infinite-inferno?tournament=608cd3f814fa0bf9f1c1f928))? Or is that loaded in another way?

Reply:
oh yes in tournament we already use it. but you now ladder is more complicated, we have clan ladder, course ladder and hero ladder 😂

Reply:
for example i think i haven't add the user click to see session/user profile logic until now?